navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java

```diff
@@ -99,9 +102,15 @@ public MapboxNavigation(@NonNull Context context, @NonNull String accessToken) {
    */
   public MapboxNavigation(@NonNull Context context, @NonNull String accessToken,
                           @NonNull MapboxNavigationOptions options) {
+    this(context, accessToken, options, false);
+  }
+
+  public MapboxNavigation(@NonNull Context context, @NonNull String accessToken,
+                          @NonNull MapboxNavigationOptions options, boolean isFromNavigationUi) {
```

Comment:
@cammace Do you think it would be cleaner to add a boolean `isFromNavigationUi` to `MapboxNavigationOptions` rather than adding a parameter to the constructor?

Reply:
@danesfeder https://github.com/mapbox/mapbox-navigation-android/pull/277/commits/1867c75d693f9f47a0fa4abae8a369edeb089391 addresses your comment.